### DISCUSSION TITLE
Getting errror - UnicodeDecodeError: 'utf-8' codec can't decode byte 0xff in position 0: invalid start byte

### DISCUSSION THREAD

**Discussion by siddhantgithub** [JoshuaC215/agent-service-toolkit#106] [General]
Hi, 

I am getting the following error when running run_service.py:

UnicodeDecodeError: 'utf-8' codec can't decode byte 0xff in position 0: invalid start byte

I am trying to run this on a windows system. 

Any help would be much appreciated.

**Comment by JoshuaC215** [comment]
Hi @siddhantgithub, 

- Are you running just the original repo or have you made any modifications?
- Can you share the full repro steps you used?

Are you able to run it through docker instead of base python? I haven't tried running it on Windows other than Docker.
